environment.go

```diff
@@ -389,21 +389,23 @@ func (e *envState) initSenders() error {
 
 	if jira := &e.settings.Jira; len(jira.GetHostURL()) != 0 {
 		sender, err = send.NewJiraLogger(&send.JiraOptions{
-			Name:     "evergreen",
-			BaseURL:  jira.GetHostURL(),
-			Username: jira.Username,
-			Password: jira.Password,
+			Name:         "evergreen",
+			BaseURL:      jira.GetHostURL(),
+			Username:     jira.Username,
+			Password:     jira.Password,
+			UseBasicAuth: true,
```

Comment:
I don't understand why we need to use basic auth? I believe that basic auth is on the deprecated list for our jira installation, and not the preferred mechanism in general. 

The initial implementation used basic auth, and then I removed it (we could look in the history, I don't rememer the exact details at the moment,) and I can't see enough of the rationale in this (or the other PR) or associated tickets to figure things out from here.